### DISCUSSION TITLE
Using my android phone as a node

### DISCUSSION THREAD

**Discussion by Mdawg09** [meshtastic/Discussions#12] [General]
Hello, I am relatively new to meshtastic, if this isn't in the right place please move it or let me know.

I'm wondering if I could use my android phone (a50) as a node over mqtt, I don't see any reason why not but then again like I said I'm relatively new to this and if this is a dumb question please feel free to let me know.

Thanks

**Comment by rcarteraz** [comment]
No, the Android app requires you to be connected to a Meshtastic device. You can then use MQTT Proxy to connect to MQTT.

**Comment by ikearat** [comment]
Last person who should reply is me however, as I understand the system, the Meshtastic device is in communication with the Internet not the phone. The phone is not the one talking to the rest of the Meshtastic network, it's using the node as a "modem" to talk to the other "modems" on the MQTT system. So the phone can't act like a Meshtastic modem by it's self otherwise there would be little use for 
MQTT where everyone would just skip the Mestastic device and go directly to other users over the Internet (see: Facebook, "X", Reddit, etc.)

Edit: rcarteraz is who you should listen too. :)

**Reply by Mdawg09** [reply]
So is there any reason why I can't use my phone as a "upload" to mqtt so I can reach my cottage for example?

**Reply by rcarteraz** [reply]
As long as you have a Meshtastic device paired to your phone, you can.

**Reply by Mdawg09** [reply]
Yes i understand that but thats not the point of my question, the idea is to not have a node but still be able to check up on my remote nodes that are connected to wifi

**Reply by ikearat** [reply]
There would have to be a "gateway" server on the Internet. The Meshtastic devices are the bridge to the Internet from non-LoRa devices. To the best of my understanding there is no gateway into the Meshtastic mesh (the mesh is independent from the Internet) other than a node configured to pass Internet mesh traffic to the Internet (MQTT). MQTT does not serve the node configuration app the node serves the configuration app. You could try making your home node "visible" using VPN and your home computer and doing remote management of your home computer. You could then operate your node(s) using the web client through your remote control arrangement. The node at your house or wherever the computer is would be your window into the configuration and then you could access your remote nodes from there. I think I created mud with my posting so if you ask, I will try to clear it up as much as I can :)
After posting this I got an idea. If you make public the IP addresses of the nodes connected to WiFi you can control them using the web utility without a remote managed VPN. The down-side is making your IP addresses public. There may be a way to tunnel to your device if you can control your home router (or the routers providing WiFi to your nodes) without exposing the rest of the machines on your LAN.

**Reply by Mdawg09** [reply]
Yeah, again it was just a thought, but thank you very much for working with me on this

**Comment by Mdawg09** [comment]
Ok thanks for the replies. I was just wondering, cause it seems like something that should work but it doesn't. 

Thank you again for the replies

**Reply by garthvh** [reply]
Would overwhelm a Lora mesh pretty easily, and the Lora mesh is the interesting part of meshtastic as WiFi and BLE are so much shorter range options.
